src/tests/test_fastod.cpp

```diff
@@ -0,0 +1,509 @@
+#include <algorithm>
+#include <filesystem>
+#include <memory>
+#include <string>
+#include <tuple>
+#include <utility>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+#include "algorithms/algo_factory.h"
+#include "algorithms/od/fastod/canonical_od.h"
+#include "algorithms/od/fastod/fastod.h"
+#include "config/names.h"
+#include "config/time_limit/type.h"
+
+namespace {
+
+namespace fs = std::filesystem;
```

Comment:
Don't forget to remove this line with corresponding include after fixing [this comment](https://github.com/Mstrutov/Desbordante/pull/355/files#r1527196967)

Reply:
Did it.